STEER/AOD/AliAODJet.h

```diff
@@ -117,16 +117,20 @@ class AliAODJet : public AliVParticle {
 
 
  private:
-    Double32_t      fBackgEnergy[2];         // Subtracted background energy
-    Double32_t      fEffectiveArea[2];       // Effective jet area used for background subtraction
-    Double32_t      fEffectiveAreaError[2];  //[0,1,10] relative error of jet areas, 10 bit precision
-    Double32_t      fNeutralFraction;        //[0,1,12] Neutral fraction between 0 and 1 12 bit precision;
-    Double32_t      fPtSubtracted[2];        //[0,0,12] pT after subtraction can be negative four momentum close to 0 in this case, 12 bit precision
-    Double32_t      fPtLeadingConstituent;   //[0,0,12] pT of leading constituent
-    UInt_t          fTrigger;                // Bit mask to flag jets triggered by a certain detector  
-    TLorentzVector* fMomentum;               // Jet 4-momentum vector
-    TLorentzVector* fVectorAreaCharged;      // jet area four momentum 
-    TRefArray*      fRefTracks;              // array of references to the tracks belonging to the jet
+    Double32_t      fBackgEnergy[2];         ///< Subtracted background energy
+    Double32_t      fEffectiveArea[2];       ///< Effective jet area used for background subtraction
+    /// relative error of jet areas, 10 bit precision
+    Double32_t      fEffectiveAreaError[2];  //[0,1,10]
```

Comment:
And this is fine too, right? I am wondering if there's a way to have this Doxygen-documented, and at the same time. I haven't, unfortunately, found anything even inside ROOT 6. They seem to just drop the documentation. I think that breaking it in two lines is probably the best, just checking...

Reply:
Hi Dario,
Yeah I also think that breaking it up in two lines is the best.
Cheers, Mike